agent/lib/src/health.dart

```diff
@@ -5,15 +5,26 @@
 import 'dart:async';
 import 'dart:io';
 
+import 'package:meta/meta.dart';
+import 'package:path/path.dart' as path;
+import 'package:file/local.dart' as local;
+import 'package:file/file.dart' as file;
+
 import 'adb.dart';
 import 'agent.dart';
 import 'utils.dart';
-import 'package:path/path.dart' as path;
 
 final RegExp _kLinuxIpAddrExp = RegExp(r'inet +(\d+\.\d+\.\d+.\d+)/\d+');
 final RegExp _kWindowsIpAddrExp =
     RegExp(r'IPv4 Address.*: +(\d+\.\d+\.\d+.\d+)\(Preferred\)');
 
+@visibleForTesting
+String Function() homeEnvVar = () {
+  String home = Platform.environment['HOME'];
```

Comment:
I know for this use case we only need to work with macOS, but maybe to avoid confusion in the future make this work for Windows as well?

Reply:
Added a check to ignore non-macOS.